code/datums/components/pinnable_accessory.dm

```diff
@@ -0,0 +1,80 @@
+/// This accessory can be pinned onto someone else
+/datum/component/pinnable_accessory
```

Comment:
this would be swag AF with detective's spy kit

Reply:
👆 This!